hail/python/hailtop/auth/auth.py

```diff
@@ -24,8 +24,8 @@ def get_userinfo(deploy_config=None):
     return async_to_blocking(async_get_userinfo(deploy_config))
 
 
-def namespace_auth_headers(deploy_config, ns, authorize_target=True):
-    tokens = get_tokens()
+def namespace_auth_headers(deploy_config, ns, authorize_target=True, *, _token_file=None):
```

Comment:
All this stuff is fully internal, I'm inclined to not use the underscore prefix. I read `_` in services code function parameters "I don't think this through fully; be careful".